libraries/AP_Compass/Compass_learn.cpp

```diff
@@ -108,10 +116,27 @@ void CompassLearn::update(void)
         if (num_samples > 30 && best_error < 50 && worst_error > 65) {
```

Comment:
shouldn't we have a limit here for the 'worst_error'? It seems like any value above 65 is acceptable

Reply:
we're not using this in-flight learning code in MatterNet

Reply:
Alright.